TombEngine/Objects/TR3/Entity/tr3_civvy.cpp

```diff
@@ -292,11 +297,13 @@ namespace TEN::Entities::Creatures::TR3
 						else
 							item.Animation.TargetState = CIVVY_STATE_IDLE;
 					}
-					else if ((creature.Enemy->IsLara() && AI.bite && AI.distance < CIVVY_ATTACK_CLOSE_PUNCH_RANGE) ||
-							 (creature.Enemy->IsCreature() && creature.Enemy->HitPoints > 0 && AI.ahead && AI.distance < SQUARE(BLOCK(0.5f))) )
+					else if ((validTarget && AI.distance < CIVVY_ATTACK_CLOSE_PUNCH_RANGE) &&
```

Comment:
I think you just need AI.bite in this case, CreatureAIInfo() use creature->Enemy to fill AI.bite and AI.ahead :)
no need to check if enemy is lara or creature.